### ISSUE TITLE
mu4e: key 'd' should be bind to mu4e-headers-mark-or-move-to-trash

### ISSUE THREAD

**Issue by gspe** [emacs-evil/evil-collection#328]
The key `d` should be bind to `mu4e-headers-mark-or-move-to-trash` and not to `mu4e-headers-mark-or-move-to-trash` as in the upstream [mu4e-headers.el](https://github.com/djcb/mu/blob/master/mu4e/mu4e-headers.el):

```
;; marking/unmarking ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
(define-key map (kbd "<backspace>")  'mu4e-headers-mark-or-move-to-trash)
(define-key map (kbd "d")            'mu4e-headers-mark-or-move-to-trash)
```
Without this key-bindings the user lose the opportunity to use this function:

```
- `no-trash-flag': If non-nil, the maildir will be added to
  `mu4e-move-to-trash-patterns' so that trashing moves the message instead of flagging.
```

**Comment by Ambrevar** [comment]
Does this apply to the mu4e-view-mode-map as well?

**Comment by gspe** [comment]
Yes this apply also to mu4e-view-mode-map
Sorry I forgot to report before

**Comment by Ambrevar** [comment]
Since I don't use mu4e, could you send a pull request?  
Thanks!

**Comment by Ambrevar** [comment]
Fixed with #329.
